packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -1132,6 +1132,54 @@ def test_app_without_port_index_label(
         self._assert_filter_regexp_for_invalid_app(
             filter_regexp, app, nginx_class, mocker, valid_user_header)
 
+    def test_app_container_networking_with_invalid_port_mapping_index_label(
```

Comment:
I did not review the tests, but thanks for writing them! :rocket: If you want to have feedback also from @vespian just let him know :-)